src/sentry/models/projectownership.py

```diff
@@ -287,6 +288,19 @@ def handle_auto_assignment(cls, project_id, event):
                 }
             )
 
+            activity = Activity.objects.filter(
```

Comment:
Idk much about the autoassignment feature, this was tricky to figure out so I hope it's right 😬 if it's not please point me towards how I can determine if an issue already has a manual assignment.